soc/xtensa/intel_adsp/ace_v1x/Kconfig.defconfig.series

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_INTEL_ACE1X
+
+config SOC_SERIES
+	string
+	default "ace_v1x"
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "intel_s1000"
+
+config SOC
+	string
+	default "intel_ace15_mtpm"
+
+config IMR_MANIFEST_ADDR
+	default 0xa1042000
+
+config MP_NUM_CPUS
+	default 3
+
+config SMP
+	default y
+
+# MTL leaves the upper mapping in the same spot as cAVS, but moves the
+# lower one inexplicably.
+config XTENSA_UNCACHED_REGION
+	default 2
+
+# Parameters for gen_isr_tables.py:
+
+config 2ND_LVL_INTR_00_OFFSET
+	default 4
+
+config MULTI_LEVEL_INTERRUPTS
+	default y
+
+config MAX_IRQ_PER_AGGREGATOR
+	default 29
+
+config NUM_2ND_LEVEL_AGGREGATORS
+	default 1
+
+config 2ND_LVL_ISR_TBL_OFFSET
+	default 9
+
+config 2ND_LEVEL_INTERRUPTS
+	default y
+
+config XTENSA_TIMER
+	default n
+
+config XTENSA_TIMER_ID
+	default 0
+
+config MTL_TIMER
+	default y
```

Comment:
This produces a warning:

```
warning: MTL_TIMER (defined at soc/xtensa/intel_adsp/ace_v1x/Kconfig.defconfig.series:58) defined without a type
```

Because nothing ever defines the MTL_TIMER kconfig.  Presumably that got dropped from the submission?  The kernel timer driver is a pretty core thing and needs to land at the same time as the platform in most cases.

Also: note that this looks like another cut-and-paste job incoming.  The cAVS and ACE/MTL hardware have identical timer interfaces modulo the address of the registers. (The all-purpose/miscellaneous "shim" block that used to be there got split up into separate blocks in ACE).  They really want to be using the same driver, not a copied version.

Reply:
Looks like these are being caught by twister in CI too; no need to fetch and build locally:

https://github.com/zephyrproject-rtos/zephyr/runs/6843591805?check_suite_focus=true (RED)

```
-- Generated device_extern.h: /__w/zephyr/zephyr/twister-out/nxp_adsp_imx8m/samples/subsys/audio/sof/sample.audio.sof/zephyr/include/generated/device_extern.h
-- Including generated dts.cmake file: /__w/zephyr/zephyr/twister-out/nxp_adsp_imx8m/samples/subsys/audio/sof/sample.audio.sof/zephyr/dts.cmake

warning: SMP_BOOT_DELAY (defined at kernel/Kconfig:859) was assigned the value 'y' but got the value
'n'. Check these unsatisfied dependencies: SMP (=n). See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_SMP_BOOT_DELAY and/or look up
SMP_BOOT_DELAY in the menuconfig/guiconfig interface. The Application Development Primer, Setting
Configuration Values, and Kconfig - Tips and Best Practices sections of the manual might be helpful
too.

warning: MTL_TIMER (defined at soc/xtensa/intel_adsp/ace_v1x/Kconfig.defconfig.series:58) defined without a type
Parsing /__w/zephyr/zephyr/Kconfig
Loaded configuration '/__w/zephyr/zephyr/boards/xtensa/nxp_adsp_imx8m/nxp_adsp_imx8m_defconfig'
Merged configuration '/__w/zephyr/zephyr/samples/subsys/audio/sof/prj.conf'
Merged configuration '/__w/zephyr/zephyr/samples/subsys/audio/sof/boards/nxp_adsp_imx8m.conf'
error: Aborting due to Kconfig warnings
CMake Error at /__w/zephyr/zephyr/cmake/modules/kconfig.cmake:290 (message):
  command failed with return code: 1
Call Stack (most recent call first):
  /__w/zephyr/zephyr/cmake/modules/zephyr_default.cmake:121 (include)
  /__w/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:51 (include)
  /__w/zephyr/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:76 (include_boilerplate)
  CMakeLists.txt:5 (find_package)
-- Configuring incomplete, errors occurred!
```

>  I'm sure it's frustrating that you're being asked to manage upstreaming for someone else's code

Ouch, what could possibly go wrong...